specification/monitor/resource-manager/microsoft.insights/stable/2018-01-01/metrics_API.json

```diff
@@ -0,0 +1,429 @@
+        }
+      },
+      "description": "A time series result type. The discriminator value is always TimeSeries in this case."
+    }
+  },
+  "parameters": {
+    "ResourceUriParameter": {
+      "name": "resourceUri",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The identifier of the resource.",
+      "x-ms-parameter-location": "method",
+      "x-ms-skip-url-encoding": true
+    },
+    "TimespanParameter": {
+      "name": "timespan",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "The timespan of the query. It is a string with the following format 'startDateTime_ISO/endDateTime_ISO'.",
+      "x-ms-parameter-location": "method"
+    },
+    "IntervalParameter": {
+      "name": "interval",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "format": "duration",
+      "description": "The interval (i.e. timegrain) of the query.",
+      "x-ms-parameter-location": "method"
+    },
+    "MetricParameter": {
+      "name": "metric",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "The name of the metric to retrieve.",
+      "x-ms-parameter-location": "method"
+    },
+    "AggregationsParameter": {
+      "name": "aggregation",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "The list of aggregation types (comma separated) to retrieve.",
+      "x-ms-parameter-location": "method"
+    },
+    "TopParameter": {
+      "name": "$top",
+      "in": "query",
+      "required": false,
+      "type": "number",
+      "format": "int32",
+      "description": "The maximum number of records to retrieve.\nValid only if $filter is specified.\nDefaults to 10.",
+      "x-ms-parameter-location": "method"
+    },
+    "OrderByParameter": {
+      "name": "$orderby",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "The aggregation to use for sorting results and the direction of the sort.\nOnly one order can be specified.\nExamples: sum asc.",
+      "x-ms-parameter-location": "method"
+    },
+    "ResultTypeParameter": {
+      "name": "resultType",
+      "in": "query",
+      "type": "string",
+      "enum": [
+        "Data",
+        "Metadata"
+      ],
+      "x-ms-enum": {
+        "name": "ResultType",
+        "modelAsString": false
+      },
+      "description": "Reduces the set of data collected. The syntax allowed depends on the operation. See the operation's description for details.",
+      "x-ms-parameter-location": "method",
+      "required": false
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    },
+	"MetricNamespaceParameter": {
```

Comment:
>MetricNamespaceParameter [](start = 2, length = 24)

Is this parameter meant for the client or the method?

Reply:
Client